packages/hardhat-core/test/internal/hardhat-network/provider/modules/eth.ts

```diff
@@ -4506,3 +4508,47 @@ describe("Eth module", function () {
     });
   });
 });
+
+describe("Eth module - special tests", function () {
```

Comment:
Created this other tests as they depend on fixture projects, and not the `PROVIDERS` array.